core/src/test/scala/integration/kafka/admin/BrokerApiVersionsCommandTest.scala

```diff
@@ -25,10 +25,12 @@ import kafka.server.KafkaConfig
 import kafka.utils.TestUtils
 import org.apache.kafka.clients.NodeApiVersions
 import org.apache.kafka.common.protocol.ApiKeys
-import org.apache.kafka.common.requests.ApiVersionsResponse
+import org.apache.kafka.test.IntegrationTest
 import org.junit.Assert.{assertEquals, assertFalse, assertTrue}
 import org.junit.Test
+import org.junit.experimental.categories.Category
 
+@Category(Array(classOf[IntegrationTest]))
```

Comment:
Discussed this offline, seems like we can simply change the traits to be abstract classes. The code compiles fine.